src/utils.rs

```diff
@@ -1,5 +1,4 @@
 use movingai::Coords2D;
-use std::cmp::{max, min};
```

Comment:
Small note here: you don't need to import them. You were comparing numbers for min/max, and all numeric primitives implement the min/max functions, so you can call them directly. I.e.:
```rs
let min = 10f64.min(5.0);
```